### PR TITLE
Use ansible facts variable instead of `rhelversion` variable

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: roles/sat6repro/tasks/main.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - name: disable all repos
   command: subscription-manager repos --disable "*"
 - name: Enable required repos
-  command: subscription-manager repos --enable rhel-{{ rhelversion }}-server-rpms --enable rhel-server-rhscl-{{ rhelversion }}-rpms --enable rhel-{{ rhelversion }}-server-satellite-{{ satelliteversion }}-rpms
+  command: subscription-manager repos --enable rhel-{{ ansible_distribution_major_version }}-server-rpms --enable rhel-server-rhscl-{{ ansible_distribution_major_version }}-rpms --enable rhel-{{ ansible_distribution_major_version }}-server-satellite-{{ satelliteversion }}-rpms
```
Comment (johnpmitsch) [inline roles/sat6repro/tasks/main.yml]: This fact is coming from the destination node (and not from the control node)?

Comment (sthirugn) [inline roles/sat6repro/tasks/main.yml]: yes it comes from the destination node.  We are already using this variable in some places in main.yml

Comment (johnpmitsch) [inline roles/sat6repro/tasks/main.yml]: great :+1:

**Comment by sthirugn** [comment]
Requesting review of this PR. This PR is open for so long.

**Pull request merged**